CHANGES.rst

```diff
@@ -3,6 +3,9 @@ Version v5.3.3 (2023-09-07)
 
 Bug Fixes
 ---------
+astropy.io.fits
```

Comment:
The change should not be written directly here, but rather add a fragment `15388.bugfix.rst` in `docs/changes/io.fits`

Reply:
See docs/changes/README.rst